index.bs

```diff
@@ -870,7 +914,7 @@ use a [=roaming authenticator=] which was originally registered with the [RP] us
 
     :   <dfn>attachment</dfn>
     ::  This member contains authenticator attachment descriptions, which are used as an additional constraint on which
-        authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation. See [[#attachment]]
+        authenticators are eligible to participate in a {{ScopedCredential/create()}} operation. See [[#attachment]]
```

Comment:
It's only passed into a `create()` operation, but I suppose it is used in the backend decisions for `get()`? Do we need to pass it into `get()` as well?